doc/guides/gps_setup.md

```diff
@@ -0,0 +1,57 @@
+---
+module: [kind=guide] gps_setup
+---
+
+# Setting up GPS
+Some quick definitions for this page.
+```
+GPS host: a computer running the GPS program in host mode
+GPS constellation: a group of GPS hosts
+```
+
+Before GPS can be used in a dimension (e.g. the Nether) a GPS host constellation needs to be set up. Traditionally this is done near the max build height for maximum range, but ender modems easily beats a normal modem's max range at any height.
+
+To give the best results, a GPS constellation needs at least four computers. One can theoretically get away with three for most GPS requests, but four changes that to 100% accuracy when the requester is within all of their ranges, provided that they are placed correctly. Having more than four GPS hosts per constellation is redundant, but it does not harm.
```

Comment:
> One can theoretically get away with three for most GPS requests,

Hrm, are you sure about this? My understanding is that you need four - three only narrows your position down to two points.